.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+### Description
+
+[Please describe what this change achieves]
+
+### Definition of Done

```

Comment:
Do doc submissions have doneness criteria beyond "is written"? If so, maybe some more details in the checklist instead?

Reply:
- [ ] Consensus that these instructions are the best path forward
- [ ] Consensus this is cruft and should be removed from the docs
- [ ] Can build the thing following these instructions, without unmentioned prior art or knowledge

What other examples can you think of?

Reply:
Sounds good to me, though I think a lot of that "consensus" gets to be whatever you want as the most active party ;-)